gui/resultstree.cpp

```diff
@@ -900,6 +904,22 @@ void ResultsTree::copyMessageId()
     }
 }
 
+void ResultsTree::copyLineNr()
+{
+    if (mContextItem) {
```

Comment:
Please use early return.

Reply:
Ok i will change it.
Is it because of performance reasons or more a stylistic thing?

Reply:
stylistic.

early returns are not used always in the GUI code but I try to refactor it so it is.